### PR TITLE
Update feedstock

### PR DESCRIPTION
**Pull request by ocefpaf**
~~Copying `geos.dll` and `geos_c.dll` to `%SCRIPTS%`.~~

### CODE REVIEW

File: recipe/bld.bat [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
-set INCLUDE=%LIBRARY_INC%;%INCLUDE%
+nmake install
+if errorlevel 1 exit 1
 
-cmake .. -G "NMake Makefiles" -DCMAKE_INSTALL_PREFIX:PATH=%LIBRARY_PREFIX% -DCMAKE_BUILD_TYPE=Release
-
-nmake install || exit 1
-
-copy ..\include\geos.h %LIBRARY_INC%\geos.h || exit 1
+copy bin\*.dll %SCRIPTS%\ || exit 1
```
Comment (ocefpaf) [inline recipe/bld.bat]: I am not sure if this will affect the packages we have. Apparently `cartopy` and `shapely` were OK with just the `.lib` files.

**Comment by gillins** [comment]
Scripts is in %PATH% so Windows should be able to find the .dlls here too. The .lib files are specified to the linker for these packages, but these are actually 'import libraries' that point to the .dll.
But why is this necessary? Seems confusing to have the dll files for GEOS in %SCRIPTS% when everything else is in %LIBRARY_BIN%

**Comment by ocefpaf** [comment]
> Seems confusing to have the dll files for GEOS in %SCRIPTS% when everything else is in %LIBRARY_BIN%

I can copy them to `%LIBRARY_BIN%`.But I notice that both `expat` and `udunits2` copy/move the `DLL` to `%SCRIPTS%`. So I am confused as well.

**Comment by ocefpaf** [comment]
I took a second look at this and even though `udunits2` and `expat` copy the DLLs to `%SCRIPTS%` [`shapely`](https://github.com/conda-forge/shapely-feedstock/blob/master/recipe/bld.bat#L5) expect `geos` DLLs to be at `%LIBRARY_BIN%`.

So I modify this PR to leave the DLLs alone.  (Maybe we should fix `expat` and `udunits2`?)

@msarahan what is continuum policy for the default channel? DLLs in `%LIBRARY_BIN%`?

**Pull request merged**